CHARTER.md

```diff
@@ -0,0 +1,108 @@
+# JSON Schema Charter
+<!-- This document is managed in the json-schema-org/community GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+## 0: Guiding Principles
+The JSON Schema project is part of the OpenJS Foundation. The JSON Schema project strives to be open and transparent as much as is possible, and wish to enable anyone to interact and engage with any area of our work.
+
+Having no structure in place usually leads to one that is informal and undocumented, making it difficult to meet our own expectations of how the TSC wish to operate. As such, the JSON Schema project define the following charter which includes aspects of the governance model to which the TSC subscribe and by which the TSC operate.
+
+## 1: Scope
+JSON Schema aims to enable the confident and reliable use of the JSON data format. It does this primarily by providing specification documents which define a declarative language that allows annotation and validation of JSON documents.
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. The JSON Schema project aims to enable these additional and emergent use cases.
+
+### 1.1: In-scope
+The scope of the JSON Schema project is split into two sections: primary and secondary concerns.
+Primary concerns are areas the JSON Schema project wish to give focus to. Secondary concerns, while remaining in scope, aren't areas of focus, and would require dedicated championing by community members to come to fruition.
+
+Primary Concerns
+- Publication of the JSON Schema standard
+  - Validation of JSON-compatible data
+  - Semantic annotation of JSON-compatible data
+  - Interoperability
+  - Extensibility
+- Critical tooling
+- Documentation
+- Test suite
+- Community
+  - Enabling schema authors
+  - Enabling implementers
+  - Engaging with industry
+  - Engaging with upstream and downstream standards and projects
+  - Communicating value
+  - Ensuring the sustainability of the project
+
+Secondary Concerns
+- Hypermedia
+- Generating JSON Schema
+- Using JSON Schema to generate
+  - Code (including types or classes)
+  - UI (including forms)
+  - Databases
+- Relational validation
+- Vocabularies registry
+
+### 1.2: Out-of-Scope
+Neither standards that the JSON Schema project uses (such as JSON and IRI) nor standards or projects that use JSON Schema (such as OpenAPI or AsyncAPI) are in scope, nor does the JSON Schema project have any control over them.
```

Comment:
This sentence seems a bit in conflict with the interoperability goal. I guess being interoperable with OpenAPI is somewhat part of the mandate?

Reply:
I would argue no. Our interoperability objective is across languages.
Say OpenAPI released an update that made some new requirement of JSON Schema... We should not bend the JSON Schema specification to match something else, which could be detrimental to many other use cases.

The point of this phrasing/section is to make clear that we do not control upstream or downstream projects. We often get people asking "can you make JSON do X?" or "can you make OpenAPI do Y?".

Of course, we are open to, have been, and will continue to collaborate and provide expert advice to upstream and downstream projects, as we are able to. But, not at the expense of the JSON Schema project itself, and not a significant amount of time.

Reply:
Thx